testing/sdk_utils.py

```diff
@@ -75,3 +76,11 @@ def is_strict_mode():
 dcos_ee_only = pytest.mark.skipif(
     is_open_dcos(),
     reason="Feature only supported in DC/OS EE.")
+
+
+# Pretty much https://github.com/pytoolz/toolz/blob/a8cd0adb5f12ec5b9541d6c2ef5a23072e1b11a3/toolz/dicttoolz.py#L279
+def get_in(keys, coll, default=None, use_default=True):
```

Comment:
The `use_default` parameter is not used at all. Could we add a basic docstring here too?

Reply:
Good point, I'll remove it and add a docstring.

Reply:
https://github.com/mesosphere/dcos-commons/pull/1855/commits/0c2e868502ed7105a1ca4736ffa99ae7f11b37c5